stock_reserve_area/views/stock_reserve_area_views.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+        <record id="view_reserve_area_form" model="ir.ui.view">
+            <field name="name">stock.reserve.area</field>
+            <field name="model">stock.reserve.area</field>
+            <field name="arch" type="xml">
+                <form string="Reserve Area">
+                    <sheet>
+                        <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
+                        <label for="name" class="oe_edit_only" />
+                        <h1><field name="name" /></h1>
+                        <group>
+                            <group>
+                                <field name="active" invisible="1" />
+                            </group>
+                            <group>
+                                <field
+                                name="company_id"
+                                groups="base.group_multi_company"
+                                options="{'no_create': True}"
+                            />
+                            </group>
+                            <group>
+                                <field
+                                name="location_ids"
+                                widget="many2many"
+                                domain="[('reserve_area_id', '=', False), ('usage', '=', 'internal')]"
```

Comment:
Why not include the view locations? Otherwise you cannot inclue, for example, the entire WH

Reply:
Also, seems better to put the domain in the field itself instead of the view.